### DISCUSSION TITLE
Will there be a `cds-combobox` component as the one available in Clarity Angular

### DISCUSSION THREAD

**Discussion by jyothi530** [vmware-clarity/core#162] [General]
Hi, Will there be a `cds-combobox` component as the one available in Clarity Angular.

**Comment by ashleyryan** [comment]
The timeline for Core is unclear at the moment, but yes, we're hoping to eventually have a combobox component. The dropdown/popover component that was released in beta for v6 is setting some of the foundation for the combobox.

**Reply by d-m-s** [reply]
## Combobox: support for custom values 
please also consider including the feature request for [custom values / free text input](https://github.com/vmware-clarity/ng-clarity/discussions/171) in the core-component.

See https://github.com/vmware-clarity/ng-clarity/discussions/171

**Reply by arcmodo** [reply]
@ashleyryan I don’t suppose there’s any update to a potential timeline for this?
